_posts/2019-05-20-foreman-community-newsletter---may-2019.md

```diff
@@ -0,0 +1,156 @@
+
+### Joining the community team
+
+At the start of March, I made a decision to join th Foreman community team.
+I have been running the community demos on youtube as you might have already
+heard and recently I have been working with the community to get the foreman
+documentation more structured and readable. I would speak more about it in the
+upcoming demos and blog about it to keep you updated on its progress. Too
+much about my work, lets see what we missed in the recent 2 months.
+
+### 1.22.0 - RC1 is ready for testing! (and RCs for Katello 3.12 too)!
+
+Work on 1.22 has been rolling on throughout the last couple of months, and it's
+time once again to ask you all to do some testing with us! 1.22 RC1 was
+released [a week ago][1_22_rc1]. This brings a type support in Parameters,
+various compute resource libraries update, fixing the broken Google Compute
+Resource, Graphql API and Smart Proxy Exposed Capabilities and Settings. Read the
+[release notes][release_notes] to know more about the new features and *ton* of fixes done in this
+release.
+We've just released [1.22.0 RC2][1_22_rc2] for round of testing, so give
+it a spin and let us know how you get on - and do [report issues][issues] as
+you find them! We also had a test week kindly organized by Lukas[lzap] in the
+simillar fashion as done in the previous release. We couldn't do this without
+the testing we get from the community, so thanks very much to you all!
+
+[1_22_rc1]: https://community.theforeman.org/t/foreman-1-22-0-rc1-is-ready-for-testing/13766
+[issues]: https://projects.theforeman.org/projects/foreman/issues
+[release_notes]: https://theforeman.org/manuals/1.22/index.html#Releasenotesfor1.22
+
+As ever, many thanks to the contributors to these releases - coders, RC testers,
+documentation writers, and bug/feature reporters. Foreman wouldn't be what it
+is without you!
+
+### Conference roundup
+
+As ever, Foreman was present at Red Hat Summit this year,
+and as usual the Foreman/Pulp booth looked incredibly busy every time the expo
+area open. The talks by Tomer and Ewoud were well received. As Tomer
+mentioned in the community demo, a lot of questions were asked around Ansible
+integration, working of foreman with Ansible Tower, Remote execution and also
+acknowledged the improvements to content view publishing, better UX for repos
+and audit. Sounds like good news to us! All the work that we have been putting
+in is being noticed :)
+
+Not to forget, we had the very awesome yellow Foreman hats distributed at the Foreman
+booth. Thanks to Tomer and Ori for handling the design and shipment of the goodies.
+
+### Community Survey analysis is now out
+
+Thanks to everyone who filled out this year's community survey - we have now
+the analysis of the report that was prepared by Greg. We had 116 replies, which is a
+little down on last year, but still more than enough data to work with. As promised, Greg has
+written us a blog. It should be up and running any time :)
```

Comment:
Greg's post is merged, I think few images are breaking and that's why it is not published on the website. @ekohl can you please take a look at it, I want to add a link to @GregSutcliffe's post in the blog :)

Reply:
You can link to the blog post just fine, and you probably should, as thats the canonical URL. It's the integration/mirroring with Discourse that is broken.